akka-cluster/src/main/scala/akka/cluster/ClusterDaemon.scala

```diff
@@ -395,10 +396,23 @@ private[cluster] class ClusterCoreDaemon(publisher: ActorRef, joinConfigCompatCh
       context.actorOf(propsWithDispatcher, name = "downingProvider")
     }
 
-    if (seedNodes.isEmpty)
-      logInfo("No seed-nodes configured, manual cluster join required")
-    else
+    if (seedNodes.isEmpty) {
+      if (isClusterBootstrapUsed)
+        logDebug("Cluster Bootstrap is used for joining")
```

Comment:
Maybe also `logInfo`?

Reply:
Bootstrap is logging plenty on info level itself so I don't think that is necessary.

Reply:
I see, makes sense then.